lib/unsupported-features/node-builtins-modules/events.js

```diff
@@ -0,0 +1,65 @@
+const { READ } = require("@eslint-community/eslint-utils")
+
+/** @type {import('../types.js').SupportVersionTree} */
+const static = {
```

Comment:
`static` is a reserved keyword and will throw in strict mode.

Reply:
😱 Good catch